### PR TITLE
feat: refactor api key usage

### LINKED ISSUE #1: Use of Non Global API
Hello,

Awesome work on this integration by the way!

However is there any chance that we don't need to use the Global API key? Not to happy security wise to be using the Global API.

Aydan

### PR DESCRIPTION
**Pull request by syphernl**
This refactors the codebase to use non-global API keys

* Replace Global API requirement with non-global
* Use the API token verification endpoint to verify credentials
* Removed the `email` field from the configuration
* Added Dutch translation
* Small code refactors

Note that these changes will no longer work with the **legacy** Global API keys and the integration will need to be re-configured.
Since they are [considered legacy](https://developers.cloudflare.com/fundamentals/api/get-started/keys/) by Cloudflare it is probably not worth the hassle to make it backwards compatible. 

I recommend releasing this as a new major release with a breaking change remark to inform users of this change.

Fixes #1

### CODE REVIEW

**Comment by syphernl** [comment]
I have tested this on my local install and it works perfectly.

File: custom_components/cloudflare_tunnel_monitor/const.py [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
+# Constantes para las claves de los datos de usuario
+CONF_API_KEY = "api_key"
+CONF_ACCOUNT_ID = "account_id"
+
+# Constantes para las etiquetas que se mostrarán en la interfaz de usuario
+LABEL_API_KEY = "API Key"
+LABEL_ACCOUNT_ID = "Account ID"
+
+# Constantes para los placeholders o valores sugeridos
+PLACEHOLDER_API_KEY = "global-api-key"
```
Comment (WhyAydan) [inline custom_components/cloudflare_tunnel_monitor/const.py:12]: @syphernl This doesn't need to be "global-api-key" anymore :)

Comment (syphernl) [reply custom_components/cloudflare_tunnel_monitor/const.py]: Good catch! I thought I had removed/replaced all references to `global` but this one slipped through. Fixed 😄

Comment (WhyAydan) [reply custom_components/cloudflare_tunnel_monitor/const.py]: I've lost count the amount of times this has happened to me ;)

**Pull request merged**